setup/generic/install.sh

```diff
@@ -63,7 +63,7 @@ then
     cat <<- EOF > /etc/rc.local
 		#!/bin/bash
 		{
-		  while ! curl -s https://raw.githubusercontent.com/marcone/teslausb/main-dev/setup/generic/install.sh
+		  while ! curl -s https://raw.githubusercontent.com/MagoKimbra/teslausb/main-dev/setup/generic/install.sh
```

Comment:
Don't change the repo

Reply:
But then here when I test it searches mine or does it go to yours and take the old files?
Then it's okay that the right name remains in yours, but now while I'm testing I need you to download the files I modified.